bdr-veeam/veeam-configure-local-backup-jobs.ps1

```
@@ -0,0 +1,242 @@
+            $OPTIONS.JobOptions.BackupWindow = $WINDOW
+
+            # Apply
+            Set-VBRJobOptions -Job $JOB -Options $OPTIONS
+            Write-Host "    [OK] Backup window applied."
+        } catch {
+            # Try Set-VBRJobSchedule as fallback
+            try {
+                Set-VBRJobSchedule -Job $JOB `
+                    -DailyOptions (New-VBRDailyOptions -DayOfWeek Monday,Tuesday,Wednesday,Thursday,Friday -Period 1) `
+                    -At "05:00"
+                Write-Host "    [OK] Schedule set (daily Mon-Fri at 5 AM)."
```

PSScriptAnalyzer (1.24.0)

[warning] File 'veeam-configure-local-backup-jobs.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)